crates/python/tests/compiler/test_quilc.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from qcs_sdk.qpu.isa import InstructionSetArchitecture
+from qcs_sdk.compiler.quilc import (
+    DEFAULT_COMPILER_TIMEOUT,
+    CompilerOpts,
+    TargetDevice,
+    QuilcError,
+    compile_program,
+    compile_program_async,
+    get_version_info,
+    get_version_info_async,
+)
+
+@pytest.fixture
+def target_device(aspen_m_3_isa: InstructionSetArchitecture) -> TargetDevice:
+    return TargetDevice.from_isa(aspen_m_3_isa)
+
+
+def test_default_compiler_timeout():
+    """Ensure this exists and is the correct type."""
+    assert type(DEFAULT_COMPILER_TIMEOUT) is float
```

Comment:
Not for this PR, but I wonder if we should consider something like [stubtest](https://mypy.readthedocs.io/en/stable/stubtest.html) to help validate that our stubs match what our package actually exports.

Reply:
There is also [stubgen](https://mypy.readthedocs.io/en/stable/stubgen.html), which would be a huge boon. Unclear if it will work with the bindings though.

Reply:
Awesome! I'm going to use stubtest for the quil-py stuff too. Will be good to have some real validation against the stubs.

I think we probably went down a similar rabbit hole with `stubgen`. I tried that and a few other tools that do something similar with no luck. It might just not be possible for bindings.